content/en/docs/contribute/participating.md

```diff
@@ -32,65 +32,61 @@ aspects of Kubernetes -- the Kubernetes website and documentation.
 
 {{% capture body %}}
 
-## Roles and responsibilities
+## Roles overview
```

Comment:
I prefer the existing language. It's good to be clear that different roles also carry responsibility.